java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture2/tasks/_1/Number1.java

```diff
@@ -0,0 +1,154 @@
+package com.zoolatech.lecture2.tasks._1;
+
+import java.util.Scanner;
+import java.lang.ArithmeticException;
+
+/*Define a class that represents a calculator.
+ *  The class should provide methods that accept another value and perform
+ *  addition, subtraction, multiplication and division operations
+ *  on a value stored in a calculator instance.
+ *  Divides by a zero should be forbidden and ignored.
+ *  The class should also provide a method to get a current value.
+ *  The class should work with both integer and double numbers (ignore roundoff errors).
+ *  Assume all operation results fit into the range of values for a current value type.
+ */
+
+public class Number1 {
+    static Scanner scanner = new Scanner(System.in);
+     static Calculator calculator = new Calculator(0);
+
+    public static void main(String[] args) {
+
+        System.out.println("1. Start");
+        Scanner scanner = new Scanner(System.in);
+
+        if (scanner.nextInt() == 1) {
+            calculations();
+        } else {
+            throw new IllegalArgumentException("Unknown" + scanner.nextInt());
+        }
+    }
+
+    public static void calculations() {
+        System.out.println("Input first number:");
+        if (scanner.hasNextInt()) {
+            integerCalc();
+        } else if (scanner.hasNextDouble()) {
+            doubleCalc();
+        }
+    }
+
+    private static void integerCalc() {
```

Comment:
`integerCalc` and `doubleCalc` have some common code that can be extracted into a common place